dev/flake.nix

```diff
@@ -15,6 +15,10 @@
         (inputs.git-hooks + /flake-module.nix)
       ];
 
+      flake.om.develop.default.readme = # md
+        ''
+          🐍 Welcome to the Python Flake Project!
```

Comment:
Welcome banners are less useful than instructions telling people what to do (such as `Run "just" to see wehat's available`)

Reply:
Sure, I guess both the banner and the informative instructions can be kept here?